### PR TITLE
Bootstrap ONLYOFFICE integration documentation

### PR DESCRIPTION
**Pull request by LeSuisse**
Part of [story #27555](https://tuleap.net/plugins/tracker/?aid=27555): display Office documents stored in the Document plugin

### CODE REVIEW

File: languages/en/user-guide/integration/onlyoffice.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+ONLYOFFICE
+==========
+
+`ONLYOFFICE <https://www.onlyoffice.com/>`_ is an office suite you can use in your browser.
+
+Integration with :ref:`document-manager`
+----------------------------------------
+
+Office documents (Word, Excel, spreadsheets, PDF…) stored in the Document Manager can be directly viewed in ONLYOFFICE.
```
Comment (nterray) [inline languages/en/user-guide/integration/onlyoffice.rst:11]:
```suggestion
Office documents (Word, Excel, spreadsheets, PDF, …) stored in the Document Manager can be directly viewed in ONLYOFFICE.
```

File: languages/en/user-guide/integration/onlyoffice.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The integration is seamless when the ONLYOFFICE integration plugin is active: clicking on a office document in the Document
```
Comment (nterray) [inline languages/en/user-guide/integration/onlyoffice.rst:12]:
```suggestion
The integration is seamless when the ONLYOFFICE integration plugin is active: clicking on an office document in the Document
```

File: languages/en/user-guide/integration/onlyoffice.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Manager will open ONLYOFFICE and load your document.
+
```
Comment (nterray) [inline languages/en/user-guide/integration/onlyoffice.rst:14]: Should we add a note to emphasize that the integration is read only?

**Review by nterray — CHANGES_REQUESTED** [review]

File: languages/en/user-guide/integration/onlyoffice.rst (hunk 1 above)
Comment (LeSuisse) [reply languages/en/user-guide/integration/onlyoffice.rst:11]: Are you sure about this? In French that is not the case and I was under the impression it was the same in English?

Comment (nterray) [reply languages/en/user-guide/integration/onlyoffice.rst:11]: No I'm not sure.

**Review by nterray — APPROVED** [review]

**Pull request merged**